### ISSUE TITLE
bug: `laravel-zero` new app-name don't rename the app

### ISSUE THREAD

**Issue by stefkes** [laravel-zero/laravel-zero#281]
Installation of Laravel Zero with the command below

`laravel-zero new TestApp`

Results in output

```
Writing lock file
Generating optimized autoload files
> @php application app:rename
Renaming the application...

 What is your application name?:
 >

  Aborted.


Script @php application app:rename handling the post-create-project-cmd event returned with error code 1
Application ready! Build something amazing.
```

No big deal, I can set the name in `config/app.php` to TestApp

However, when I then try create a Command using the command below

`php TestApp make:command DoStuff`

I get

`Could not open input file: TestApp`

No idea if the issues are related. Any idea what's causing this? 

This is running `PHP 7.3.15 (cli)` on OSX Mojave

**Comment by owenvoke** [comment]
This is a known issue with how Composer works. I guess it probably might be good to remove it from the `post-install` script...

Changing the app name in the configuration doesn't change the binary, you can run `php application app:rename {name}` to change your application binary. :+1:

**Comment by nunomaduro** [comment]
Actually also had this problem last week. Lets see if someone have a solution for this.

**Comment by owenvoke** [comment]
This should be resolved in v2.x of Composer, as mentioned in https://github.com/laravel-zero/laravel-zero/issues/184#issuecomment-597047595, however no idea when v2.x is releasing...

Commands that require user interaction don't seem to work on v1 of Composer.

**Comment by stefkes** [comment]
If composer 2 isn't available for anyone then the installation process has been broken for everyone?

**Comment by owenvoke** [comment]
I mean, that feature has been in Laravel Zero since v0.4.0, so it must have worked originally? 🤔

**Comment by nunomaduro** [comment]
Yeah, it was working before.

**Comment by jimmyt2003** [comment]
is there a workaround for this for now?

**Comment by owenvoke** [comment]
Currently not really as far as I can see... I think it's probably worth removing this to be honest (what are your thoughts @nunomaduro?) because it throws an error on first use which doesn't look good. 😕

**Comment by nunomaduro** [comment]
Let's see if we can get someone to fix this for us: https://twitter.com/LaravelZero/status/1247211861751271427?s=20.

**Comment by simonardejr** [comment]
Just made a [PR](https://github.com/laravel-zero/installer/pull/8)

**Comment by owenvoke** [comment]
I've just released v2.5.0 of the installer which resolves this issue. 👍🏻 Please note, going forward the Laravel Zero installer will be marked as deprecated (although it will continue to work).

It's now preferred to use the following:

```bash
composer create-project --prefer-dist --ask laravel-zero/laravel-zero
```

And manually rename the project after with `php application app:rename [name]`.

**Comment by owenvoke** [comment]
I've just released v2.5.0 of the installer which resolves this issue. 👍🏻 Please note, going forward the Laravel Zero installer will be marked as deprecated (although it will continue to work).

It's now preferred to use the following:

```bash
composer create-project --prefer-dist --ask laravel-zero/laravel-zero
```

And manually rename the project after with `php application app:rename [name]`.
